ui/scss/component/_form-field.scss

```diff
@@ -459,3 +460,40 @@ fieldset-section {
     }
   }
 }
+
+.date-picker-input {
```

Comment:
Somewhere the icon color needs to be set. Right now it's always black, which is too dark when the dark theme is selected

Reply:
Ah good catch, thanks @seanyesmunt will check it out.

Reply:
How about something like this

![image](https://user-images.githubusercontent.com/1719111/109055968-9c1de200-76be-11eb-8f54-a44424eb9d66.png)

And hover (my mouse is on X)

![image](https://user-images.githubusercontent.com/1719111/109055997-a4761d00-76be-11eb-9506-4751605c38d5.png)

And dark

![image](https://user-images.githubusercontent.com/1719111/109056106-c66f9f80-76be-11eb-95dc-7c34cfae5157.png)

Reply:
The buttons should be white. Just like the text

Reply:
Hello @seanyesmunt , sorry it took me this long, here are the changes:

Dark mode
![image](https://user-images.githubusercontent.com/1719111/109534782-77e54b00-7a9a-11eb-9883-9d6937f00183.png)

Light mode
![image](https://user-images.githubusercontent.com/1719111/109534819-80d61c80-7a9a-11eb-822f-2326678a42c1.png)

I'm not sure though if implementing it using

```
[theme='dark'] {
    ...
}
```

is correct or a pattern you guys use since I haven't seen it (only in `dark.scss`). Please let me know if I should change it. Thanks!